gradle-plugin/doc/src/main/asciidoc/inc/_groovy-configuration.adoc

```diff
@@ -115,6 +94,57 @@ image name (either its alias or full name) or it can be a comma separated list w
 which doesn't refer an image in the configuration will be ignored.
 | `jkube.image.filter`
 
+
+ifeval::["{goal-prefix}" == "oc"]
+|*forcePull*
+|
+Applicable only for OpenShift, S2I build strategy.
+
+While creating a BuildConfig, By default, if the builder image specified in the
+build configuration is available locally on the node, that image will be used.
+
+Using forcePull will override the local image and refresh it from the registry the image stream points to.
+
+|`jkube.build.forcePull`
+endif::[]
+
+| [[image-pull-policy]] *imagePullPolicy*
+a| Specify whether images should be pull when looking for base images while building or images for starting.
+This property can take the following values (case insensitive):
+
+* `IfNotPresent`: Automatic download any missing images (default)
+* `Never` : Automatic pulling is switched off always
+* `Always` : Pull images always even when they already exist locally.
+
+By default a progress meter is printed out on the console, which is omitted when using Maven in batch mode (option `-B`). A very simplified progress meter is provided when using no color output (i.e. with `-Djkube.useColor=false`).
+| `jkube.docker.imagePullPolicy`
+
+| *logDate*
+| The date format to use when logging messages from Docker. Default is `DEFAULT` (`HH:mm:ss.SSS`)
+| `jkube.docker.logDate`
+
+| *logStdout*
+| Log to stdout regardless if log files are configured or not. Default is `false`.
+| `jkube.docker.logStdout`
+
+| *logFollow*
+| Get follow logs for your application inside Kubernetes.
+
+  Defaults to `true`.
+| `jkube.log.follow`
+
+| *logContainer*
+| Get logs of some specific container inside your application Deployment.
+
+  Defaults to `null`.
+| `jkube.log.container`
+
+| *logPod*
+| Get logs of some specific pod inside your application Deployment.
+
+  Defaults to `null`.
+| `jkube.log.pod`
+
```

Comment:
These properties don't match what you declared in the Extension, please update the extension so that the properties match these.

Reply:
oh, this looks like side effect of copy pasting log related properties into gradle docs. I'll update this to only have properties added in KubernetesExtension